client/utils/use-alipay-currencies.js

```diff
@@ -0,0 +1,93 @@
+import { useContext } from '@wordpress/element';
+import UpeToggleContext from '../settings/upe-toggle/context';
+import { useAccount } from 'wcstripe/data/account';
+
+export const useAliPayCurrencies = () => {
+	const { isUpeEnabled } = useContext( UpeToggleContext );
+	const { data } = useAccount();
+
+	let upeCurrencies = [];
+
+	if ( ! isUpeEnabled ) {
+		return [
+			'AUD',
+			'CAD',
+			'CNY',
+			'EUR',
+			'GBP',
+			'HKD',
+			'JPY',
+			'MYR',
+			'NZD',
+			'USD',
+		];
+	}
+
+	switch ( data?.account?.country ) {
+		case 'AUS':
```

Comment:
I couldn't enable AliPay on my Australian stripe account even though it had a support currency AUD and I found the issue is with this mapping function. My Australian Stripe account has the data.account.country value set to `AU` not `AUS`, I also connected a Canadian account and the data.account.country was `CA`, not `Canada`.

Has something changed since we populated this switch statement and do we need to update these switch cases to use the country codes instead?

From my testing the following account.country values need to be updated:
- `AUS` -> `AU`
- `Canada` -> `CA`
- `HongKong` -> `HK`
- `Japan` -> `JP`
- `Malaysia` -> `MY`
- `Singapore` -> `SG`

Reply:
It was a silly mistake on my end. I used country code for some and full country name for some cases :woman_facepalming: 

Thanks @mattallan. Fixed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2870/commits/686f6c8d119b0f87e2a9a364eb9ef3c5e85ebb2e